dll/win32/shell32/shfldr.h

```diff
@@ -49,6 +49,8 @@ For Details See:
 
 #define GET_SHGDN_FOR(dwFlags)         ((DWORD)dwFlags & (DWORD)0x0000FF00)
 #define GET_SHGDN_RELATION(dwFlags)    ((DWORD)dwFlags & (DWORD)0x000000FF)
+#define IS_SHGDN_FOR_PARSING(flags) ( ((flags) & (SHGDN_FORADDRESSBAR | SHGDN_FORPARSING)) == SHGDN_FORPARSING)
+#define IS_SHGDN_DESKTOPABSOLUTEPARSING(flags) ( ((flags) & (SHGDN_FORADDRESSBAR | SHGDN_FORPARSING | 0xFF)) == SHGDN_FORPARSING)
```

Comment:
What is the FF part? Is it like the bitmask used in GET_SHGDN_RELATION?

Reply:
Yes, it's the "RELATION" bits. This macro will be true when somebody asks for [`SIGDN_DESKTOPABSOLUTEPARSING`](https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/ne-shobjidl_core-sigdn).

 > Returns the parsing name relative to the desktop. This name is not suitable for use in UI.